### DISCUSSION TITLE
How to check if the file exists in node.js using fs.access?

### DISCUSSION THREAD

**Discussion by superdev1218999** [community/community#81821] [Codespaces]
### Select Topic Area

Question

### Body

I'm trying to check if the file exists in node.js using fs.existsFile.
But it isn't supported in new version of Node.js.
So I'm going to use fs.access.
How can I use it to do it?

**Comment by lightsuperred** [comment]
You can solve this issue using the fs.access.

`const fs = require('node:fs');
const fileName = 'example.json';
try {
fs.accessSync(fileName, fs.constants.F_OK);
console.log('File exists...');
} catch (err) {
console.log('File doesn't exist...');
}
`

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
